src/main/resources/wa-task-configuration-employment-et_englandwales.dmn

```diff
@@ -446,7 +450,12 @@ You can also [Reply to the Referral](/cases/case-details/${[CASE_REFERENCE]}/tri
           <text>"title"</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_1j6rfz0">
-          <text>LATEST_REVIEW_REFERRAL_RESPONSE_TASK_TITLE</text>
+          <text>if (LATEST_REVIEW_REFERRAL_RESPONSE_TASK_TITLE != null) then
+  LATEST_REVIEW_REFERRAL_RESPONSE_TASK_TITLE
+else if (taskAttributes != null and taskAttributes.name != null) then
+  taskAttributes.name
+else
+  null</text>
```

Comment:
Rather than null which can cause the task configuration issue, make a 'default' name like _**Review Referral Response**_